aep/general/0193/aep.md.j2

```diff
@@ -1,5 +1,161 @@
+the service. You **should** use the error response structure below which comes
+from the Problem Details for HTTP APIs [RFC 9457].
+
+### Structure
+
+An error response **should** contain the following fields:
+
+| Name     | Type    | Required | Description                                                                                                                                                                                                                      |
+| -------- | ------- | -------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| type     | string  | Y        | A URI reference [URI] that identifies the problem type. If the type URI is a locator (e.g., those with an "http" or "https" scheme), dereferencing it SHOULD provide human-readable documentation for the problem type.          |
+| status   | integer |          | The HTTP status code that best describes the type of problem detected.                                                                                                                                                           |
+| title    | string  |          | A human-readable description of the problem. Messages are likely to be logged in plain text and **must not** include information about a specific occurrence nor any sensitive data like PII. Information about specific occurrences should be part of `metadata`. |
+| detail   | string  |          | A human-readable explanation specific to this occurrence of the problem. This **may** contain PII and **should not** be logged.                                                                                                                                                        |
+| instance | string  |          | A unique identifier for the specific occurrence of the problem.                                                                                                                                                                  |
+
+An error response may contain additional fields appropriate for a specific
+error type.
+
+The error title and detail fields **should** help a reasonably technical user
+_understand_ and _resolve_ the issue, and **should not** assume that the user
+is an expert in your particular API. Additionally, the error title and detail
+**must not** assume that the user will know anything about its underlying
+implementation.
+
+The error detail **should** be brief but actionable. Any extra information
+**should** be provided in additional properties. If even more information is
+necessary, you **should** provide a link where a reader can get more
+information or ask questions to help resolve the issue.
+
+The error detail field
+
+- is a developer-facing, human-readable "debug message"
+- both explains the error and offers an actionable resolution to it
```

Comment:
```suggestion
- both explains the error and offers an actionable resolution to it.
- value may significantly change over time for the same error, and should not be string-matched by any clients to determine the error.
```

I think it's important to note that human-readable error messages may change, and they are not within the ste of backwards-compatible guarantees. This was a pretty big hangup at Google, with users relying on these messages to extract more information about the error.